rpc/documentation/api.md

```diff
@@ -1398,6 +1398,8 @@ transaction was seen.
   - `bool internal`: Whether the output pays to an address derived from the
     account's internal key series.  This often means the output is a change
     output.
+  - `int64 amount`: The amount for this output.
+  - `string address`: The address that this output originated from.
```

Comment:
no, outputs don't originate from addresses.  they pay to addresses (actually scripts, but usually it's an address).

There also needs to be a note here that, since not all received output scripts may be parsable as an address, this field may be null.

While here, it would be a good idea to also include the output script.  In case the script is not parsable as an address, clients can still inspect the scripts.